src/milia/api/project.cljc

```diff
@@ -97,6 +97,14 @@
   (let [url (make-url "projects" id "star")]
     (parse-http :delete url)))
 
+(defn toggle-star
+  "Toggle between starred and unstarred for a user's project"
+  [id star? callback]
+  (let [url  (make-url "projects" id "star")]
+    (parse-http (if star? :post :delete)
```

Comment:
@pld did not want to alter the original functions(in case anyone is using them) but as you say adding optional `callback` arg is the best option rather than duplicating